Tests/SwiftLintFrameworkTests/LintOrAnalyzeOptionsTests.swift

```diff
@@ -0,0 +1,71 @@
+@testable import SwiftLintFramework
+import XCTest
+
+internal typealias Leniency = (strict: Bool, lenient: Bool)
```

Comment:
Perhaps we can move this into the productive code?

Reply:
Interesting - let me try that